CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,45 @@
-# Code of conduct
+# Community Code of Conduct
```

Comment:
Would leave it as _Code of Conduct_.

Reply:
It is named like that because below there is already a section called `Code of Conduct`

Reply:
I also had doubts about the `community` part. It is called so because it belongs to the `community` repo - I checked other repos and the files there are simply called `Code of Conduct`.  On the other hand, luigi is now separate and has its own community (in a way) so we may leave it this way.

Reply:
Ok, then leaving it as it is